### PR TITLE
Fix single name variables on controller folder

### PR DESCRIPTION
**Pull request by david-vega**
Use **intention-revealing** names in variables helps the intentions of the author and readability. It can arguably avoid mistakes.

### CODE REVIEW

File: app/controllers/concerns/rate_limit_headers.rb [hunk 1]
```diff
@@ -58,7 +58,7 @@ def rate_limit_reset
   end
 
   def api_throttle_data
-    most_limited_type, = request.env['rack.attack.throttle_data'].min_by { |_, v| v[:limit] - v[:count] }
+    most_limited_type, = request.env['rack.attack.throttle_data'].min_by { |_key, value| value[:limit] - v[:count] }
```
Comment (petrokoriakin1) [inline app/controllers/concerns/rate_limit_headers.rb:61]: ohhh, please...

```suggestion
    most_limited_type, = request.env['rack.attack.throttle_data'].min_by { |_key, value| value[:limit] - value[:count] }
```

**Review by petrokoriakin1 — CHANGES_REQUESTED** [review]
Looks super reasonable. Did you try to test your code? It may be a good idea to cover changes with specs to make sure nothing breaks.

**Comment by petrokoriakin1** [comment]
hmmm... `build-and-test` takes more than 3 hours... It's a bit scary to not have fast feedback on PRs

**Comment by ClearlyClaire** [comment]
> hmmm... `build-and-test` takes more than 3 hours... It's a bit scary to not have fast feedback on PRs

It usually returns in under 10 minutes, but it since to not have run at all this time. Not sure why.

**Comment by petrokoriakin1** [comment]
> > hmmm... `build-and-test` takes more than 3 hours... It's a bit scary to not have fast feedback on PRs
> 
> It usually returns in under 10 minutes, but it since to not have run at all this time. Not sure why.

let me check it after my current task.

**Review by petrokoriakin1 — APPROVED** [review]
Thank you for the effort. I'm hitting approve here mainly to dismiss my prev review with `changes requested`. I'm still new to the mastodon processes and would rather ask for a code review from someone who understands things better.

**Review by ClearlyClaire — APPROVED** [review]
The changes look sound, but honestly I think most of these variables were pretty obvious from context.

**Pull request merged**